### ISSUE TITLE
Multiphp seams broken on Debian9 and Debian 10

### ISSUE THREAD

**Issue by JOduMonT** [hestiacp/hestiacp#640]
## After installing with

`bash hst-install.sh -a no -n yes -w yes -o yes -v yes -j no -k yes -m yes -g yes -x yes -z yes -c yes -t yes -i yes -b yes -q no -d yes -f no -y yes`

on a fresh Debian 9 and a fresh Debian 10 I open this issue because

1. I have this error
2. I'm unable to find multiple php version on my host

```
Now installing Hestia Control Panel and all required dependencies.
NOTE: This process may take 10 to 15 minutes to complete, please wait... 
/
(*) Configuring system settings...
(*) Configuring Hestia Control Panel...
(*) Generating default self-signed SSL certificate...
(*) Adding SSL certificate to Hestia Control Panel...
(*) Configuring NGINX...
hst-install-debian.sh: line 1211: /usr/local/hestia/bin/v-add-web-php: No such file or directory
hst-install-debian.sh: line 1211: /usr/local/hestia/bin/v-add-web-php: No such file or directory
hst-install-debian.sh: line 1211: /usr/local/hestia/bin/v-add-web-php: No such file or directory
hst-install-debian.sh: line 1211: /usr/local/hestia/bin/v-add-web-php: No such file or directory
hst-install-debian.sh: line 1211: /usr/local/hestia/bin/v-add-web-php: No such file or directory
(*) Configuring PHP-FPM...
hst-install-debian.sh: line 1217: /usr/local/hestia/bin/v-add-web-php: No such file or directory
(*) Configuring PHP...
(*) Configuring Vsftpd server...
(*) Configuring MariaDB database server...
(*) Installing phpMyAdmin version v4.9.2...
(*) Configuring PostgreSQL database server...
ALTER ROLE
(*) Configuring Bind DNS server...
(*) Configuring Exim mail server...
(*) Configuring Dovecot POP/IMAP mail server...
(*) Installing ClamAV anti-virus definitions...|WARNING: Ignoring deprecated option DetectBrokenExecutables at /etc/clamav/clamd.conf:33

(*) Configuring SpamAssassin...
(*) Configuring Roundcube webmail client...
(*) Configuring fail2ban access monitor...
# Warning: iptables-legacy tables present, use iptables-legacy-save to see them
hst-install-debian.sh: line 1773: /usr/local/hestia/bin/v-change-sys-theme: No such file or directory
```
## ref:

- [1211](https://github.com/hestiacp/hestiacp/blob/master/install/hst-install-debian.sh#L1206-L1212)

**Comment by ScIT-Raphael** [comment]
MultiPHP is working properly, please get the installer from release, not the master branch. You can find the link in our docs or readme.

**Comment by JOduMonT** [comment]
with Debian 9 I follow this
https://www.hestiacp.com
wget https://raw.githubusercontent.com/hestiacp/hestiacp/release/install/hst-install.sh
and I had the same error

**Comment by ScIT-Raphael** [comment]
Please try to download the file directly: https://raw.githubusercontent.com/hestiacp/hestiacp/release/install/hst-install-debian.sh

v-add-web-php is a function that will be shipped with 1.1.0 (current master branch). It should download you the installer from release branch, just checked the files, there was no change for the last 2 months.
